app/__init__.py

```diff
@@ -0,0 +1,18 @@
+import os
+from flask import Flask, request, jsonify
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+
+app.config.from_object(os.environ['APP_SETTINGS'])
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+
+
+from app.models.item import Item
+from app.controllers.items import ItemRoutes
+from app.controllers.welcome import WelcomeRoutes
+
```

Comment:
This approach works, but as the app scales up this list of models/controllers we're adding is going to increase drastically. May need to find a better way to structure this.